core/arch/aarch64/proc.c

```diff
@@ -52,6 +52,9 @@ proc_init_arch(void)
                              /* icache_line_size= */ NULL)) {
```

Comment:
Putting Derek's PR comment in a thread:

- We also need changes to drutil_insert_get_mem_addr to actually align the address
- We should invoke https://github.com/DynamoRIO/dynamorio/blob/master/clients/drcachesim/tracer/raw2trace.h#L1486 only for old traces

Reply:
I've implemented the back-alignment instructions in `drutil_insert_get_mem_addr_arm()`.
clients like `memval_simple.c` and `memtrace_simple.c` seem to be fine but the `drcachesim` tests are failing with:
```
Usage error: meta-instr faulted?  must set translation field and handle fault! (/path/to/dynamorio/core/translate.c, line 1016)
```
Any idea what this could be?
It's just 3 instructions to back-align. Possibly a translation cache issue?

Reply:
If you can repro locally (or I guess tmate on GA CI), seeing which instruction faulted in the log is usually the easiest way to understand something like this.